src/main/java/it/gov/pagopa/payhub/activities/activity/exportflow/ExportFileExpirationHandlerActivity.java

```diff
@@ -0,0 +1,15 @@
+package it.gov.pagopa.payhub.activities.activity.exportflow;
+
+import io.temporal.activity.ActivityInterface;
+import io.temporal.activity.ActivityMethod;
+
+/**
+ * Interface for the ExportFileExpirationHandlerActivity.
+ * Handles the expiration of an ExportFile identified by the provided ID.
+ */
+@ActivityInterface
+public interface ExportFileExpirationHandlerActivity {
+
+  @ActivityMethod
+  void handleExpiration(Long id, String codError);
```

Comment:
what's the meaning of the codError parameter? as described by the javadoc, you are handling the expiration of the ExportFileId... Thus why the codError?
```suggestion
  void handleExpiration(Long errorFileId, String codError);
```